spec/models/long_read_qc_result_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe LongReadQcResult, type: :model do
+  describe '#create' do
+    it 'errors without a labware_barcode' do
+      errored = false
+      begin
+        create :long_read_qc_result, labware_barcode: nil
+      rescue StandardError
+        errored = true
+      end
+      expect(errored).to eq(true)
+    end
+
+    it 'errors without a sample_id' do
+      errored = false
+      begin
+        create :long_read_qc_result, sample_id: nil
+      rescue StandardError
+        errored = true
+      end
+      expect(errored).to eq(true)
```

Comment:
expect{create :long_read_qc_result, sample_id: nil}.to raise_error(...)

Reply:
Ok, I see this was coming from lighthouse_sample_spec; I think that test was not completely right and I wouldnt use it as a source. Have a look at other tests like https://github.com/sanger/unified_warehouse/blob/develop/spec/models/qc_result_spec.rb or https://github.com/sanger/unified_warehouse/blob/develop/spec/models/study_spec.rb